src/main/java/racingcar/domain/car/Cars.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.domain.car;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import racingcar.util.RandomGenerator;
+import racingcar.view.input.error.InputError;
+import racingcar.view.input.error.InputIllegalArgumentException;
+
+public class Cars {
+
+    private static final String CARS_ARE_EMPTY = "차가 존재하지 않습니다.";
+    public static final String DISTANT_MARK = "-";
+
+    private final List<Car> cars = new ArrayList<>();
+
+    public Cars(List<String> carNames) {
+        validateCanRace(carNames);
+
+        for (String name : carNames) {
+            cars.add(new Car(name));
+        }
+    }
+
+    private void validateCanRace(List<String> names) {
+        if (names.size() < 2) {
+            throw new InputIllegalArgumentException(InputError.NOT_EXCEED_MINIMUM_AMOUNT);
+        }
+    }
+
+    public List<String> findWinners() {
+        int maxDistance = findMaxDistance();
+
+        return cars.stream()
+                .filter(car -> car.getDistance() == maxDistance)
+                .map(Car::getName)
+                .collect(Collectors.toList());
```

Comment:
이 부분은
```suggestion
                .toList();
```
로 바꿀 수 있을것 같아요!

Reply:
오......
`Collectors`를 안써도 되어 신기해 찾아보니 Java16부터 추가된 기능이군요...!! 덕분에 좋은걸 배워가네요ㅜㅜ 감사합니다! 🥰